src/main/java/seedu/address/logic/parser/AddCommandParsers/AddIssueCommandParser.java

```diff
@@ -0,0 +1,50 @@
+package seedu.address.logic.parser.AddCommandParsers;
```

Comment:
Should we package this (and the rest) into addcommand package to be in line with the Commands?

Reply:
Oh so you want to call the package just "AddCommand"? I felt that will be a little confusing as it contains parsers so AddCommandParsers felt like a more specific name. What do you think?

Reply:
Oh I thought AddCommandParser was the class name haha. Nevermind this is fine.